inspector/src/components/actionTabs/tabs/performanceViewer/performanceViewerSidebarComponent.tsx

```diff
@@ -21,6 +24,18 @@ export const PerformanceViewerSidebarComponent = (props: IPerformanceViewerSideb
     const [metadataCategoryChecked, setMetadataCategoryChecked] = useState<Map<string, number>>();
     // List of ordered categories
     const [metadataCategories, setMetadataCategories] = useState<string[]>();
+    // Min/Max/Current values of the ids
+    const [valueMap, setValueMap] = useState<Map<string, IPerfMinMax>>();
+
+    useEffect(() => {
+        const observer = (props: IVisibleRangeChangedObservableProps) => {
+            setValueMap(props.valueMap);
+        }
+        onVisibleRangeChangedObservable?.add(observer);

```

Comment:
nit - instead of using `?` (because it technically can be undefined, this is right), you can check at the beginning of the function if the `onVisibleRangeChangedObservable` is there, and return if it isn't.
This way you don't create an unneeded observer as well.

Reply:
Perfect, did that :D